11.0.1 (20B50) - No password Popup appear

When testing the script on 11.0.1 no popups will be shown and fails on password.
Works in 10.15.7 without any issues.

From Jamf policy log:
Script result: [WARNING] This script has not been tested on macOS Big Sur. Use at your own risk.
Alerting user mikael.sigemo about incoming password prompt...
Prompting mikael.sigemo for their Mac password...
Prompting mikael.sigemo for their Mac password (attempt 2)...
Prompting mikael.sigemo for their Mac password (attempt 3)...
Prompting mikael.sigemo for their Mac password (attempt 4)...
Prompting mikael.sigemo for their Mac password (attempt 5)...
[ERROR] Password prompt unsuccessful after 5 attempts. Displaying "forgot password" message...

--- a/reissue_filevault_recovery_key.sh
+++ b/reissue_filevault_recovery_key.sh
@@ -11,8 +11,8 @@
 #                   be deployed in order for this script to work correctly.
 #          Author:  Elliot Jordan <elliot@elliotjordan.com>
 #         Created:  2015-01-05
-#   Last Modified:  2020-11-02
-#         Version:  1.9.7
+#   Last Modified:  2020-12-04
+#         Version:  1.9.8
 #
 ###
 
@@ -161,7 +161,7 @@ USER_ID=$(/usr/bin/id -u "$CURRENT_USER")
 if [[ "$OS_MAJOR" -eq 10 && "$OS_MINOR" -le 9 ]]; then
     L_ID=$(/usr/bin/pgrep -x -u "$USER_ID" loginwindow)
     L_METHOD="bsexec"
-elif [[ "$OS_MAJOR" -eq 10 && "$OS_MINOR" -gt 9 ]]; then
+elif [[ "$OS_MAJOR" -eq 11 ]] || [[ "$OS_MAJOR" -eq 10 && "$OS_MINOR" -gt 9 ]]; then
     L_ID=$USER_ID
     L_METHOD="asuser"
 fi

